antarest/core/cache/business/redis_cache.py

```diff
@@ -0,0 +1,51 @@
+import json
+from typing import Optional, Any, List
+import redis
+from pydantic import BaseModel
+from redis.client import Redis
+from antarest.core.config import RedisConfig
+from antarest.core.interfaces.cache import ICache
+
+
+class RedisCacheElement(BaseModel):
+    duration: int
+    data: Any
+
+
+class RedisCache(ICache):
+    def __init__(
+        self, config: RedisConfig, redis_client: Optional[Redis] = None
+    ):
+        self.redis = (
+            redis_client
+            if redis_client is not None
+            else redis.Redis(host=config.host, port=config.port, db=0)
+        )
```

Comment:
we should extract this from (Eventbus), so that the redis_client is shared between services that needs it.